packages/analytics-js-integrations/__tests__/integrations/Amplitude/amplitude.test.js

```diff
@@ -62,4 +76,14 @@ describe('Amplitude init tests', () => {
     // eslint-disable-next-line no-underscore-dangle
     expect(typeof window.amplitude).toBe('object');
   });
+  test('Testing init call of Amplitude with EU residency set', () => {
+    const amplitude = new Amplitude(
+      amplitudeEUConfig,
+      { loglevel: 'debug', loadIntegration: true },
+      destinationInfo,
+    );
+    amplitude.init();
+    // eslint-disable-next-line no-underscore-dangle
+    expect(typeof window.amplitude).toBe('object');
```

Comment:
can't we get the properties & assert `serverZone` = `EU` ?
Otherwise, this test case is not of much help, it only checks whether the window object got initialised but not with the required values